rsync_to_rdisc.py

```diff
@@ -286,17 +297,24 @@ def run_vcf_upload(vcf_file, vcf_type, run):
 
 def upload_gatk_vcf(run, run_folder):
     run = '_'.join(run.split('_')[:4])  # remove projects from run.
+    upload_successful = True
     upload_result = []
+
     for vcf_file in glob.iglob("{}/single_sample_vcf/*.vcf".format(run_folder)):
         output_vcf_upload = run_vcf_upload(vcf_file, 'VCF_FILE', run)
         if output_vcf_upload:
             upload_result.extend(output_vcf_upload)
-    return upload_result
+
+    if any(['error' in msg.lower() for msg in upload_result]):
+        upload_successful = False
+
+    return upload_successful, upload_result
```

Comment:
These lines are the same as lines 346-350. 
Is it possible to make a function; which checks for errors and determine upload_successful?
You could call the function here and in `upload_exomedepth_vcf` instead of duplicating the code.

What are your thoughts?

Reply:
Agree, fixed :)